geth/jail/jail.go

```diff
@@ -119,15 +119,23 @@ func (jail *Jail) Call(chatID string, path string, args string) string {
 
 	// Due to the new timer assigned we need to clone existing cell to allow
 	// unique cell runtime and eventloop context.
-	cellCopy, err := cell.Copy()
-	if err != nil {
-		return makeError(err.Error())
-	}
+	// cellCopy, err := cell.Copy()
+	// if err != nil {
+	// 	return makeError(err.Error())
+	// }
 
 	// isolate VM to allow concurrent access
-	vm := cellCopy.CellVM()
+	vm := cell.CellVM()
 	res, err := vm.Call("call", nil, path, args)
 
+	// Due to the new event loop provided by ottoext.
+	// We need to ensure that all possible calls to internal setIntervals/SetTimeouts/SetImmediate
+	// work by lunching the loop.Run() method.
+	// Needs to be done in a go-routine.
+	// TODO(influx6): Figure out if we should do a throw Exception for returned error
+	// from loop.Run() call.
+	go cell.CellLoop().Run()
```

Comment:
After discussion in slack:

We could fork it and it may be a good decision. For now let's decide to leave it as-is. We may face problems with it in case many setInterval's run but it's the choice and responsibility of the user to run them.
If we face any problems with it, we'll either fork as you proposed, or make some time limits for execution, for example.
Please, add a comment warning that we can have a goroutine leakage in this place in case of improper use.